modules/tinymce/src/core/main/ts/fmt/ExpandRange.ts

```diff
@@ -210,12 +221,9 @@ const findParentContainer = (
   return container;
 };
 
-const expandRng = (
-  editor: Editor,
-  rng: Range,
-  format,
-  includeTrailingSpace: boolean = false
-) => {
+const isSelfOrParentBookmark = (container: Node) => isBookmarkNode(container.parentNode) || isBookmarkNode(container);
```

Comment:
Can `container.parentNode` be undefined here?

Reply:
It could be in specific edge cases, however this was just moved from a few lines below (see below). As such `isBookmarkNode` should be handling that from the looks (outside the scope of this anyways).
```
if (isBookmarkNode(startContainer.parentNode) || isBookmarkNode(startContainer)) {
```